app/views/ProjectEdit/Framework/index.tsx

```diff
@@ -156,37 +175,22 @@ function ProjectFramework(props: Props) {
     } = useForm(schema, defaultFormValue);
 
     const delayedValue = useDebouncedValue(value);
-
-    useEffect(() => {
-        setOffset(0);
-        setFrameworkList([]);
-    }, [delayedValue]);
-
-    const queryForFrameworks = useMemo(() => ({
-        fields: ['id', 'title', 'created_at'],
-        offset,
-        order: 'created_at',
-        limit: 10,
+    const analysisFrameworkVariables = useMemo(() => ({
+        isCurrentUserMember: delayedValue.relatedToMe === 'true' ? true : undefined,
```

Comment:
```suggestion
        isCurrentUserMember: delayedValue.relatedToMe,
```
Does this not work?

Reply:
@AdityaKhatri ,

To return ALL the analytical frameworks, relatedToMe should be neither true nor false. 

True returns **only** the AFs belonging to me .
False returns **only** the AFs that do not belong to me.

Reply:
Yes, and by default  relatedToMe is false or undefined? If its undefined then there shouldn't be any issue right?